From 26860c2e7fdeb96e42431ac85ba83910128e197e Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 5 Jul 2018 12:01:50 +0800
Subject: [PATCH 4264/5242] MLK-18605-11 ARM64: dts: imx8mm: add address and
 size cells for 'mipi_dsi'

commit  40d5c8217cdd16ea1d0bc91a6eaa874cf348721a from
https://source.codeaurora.org/external/imx/linux-imx.git

Add a different '#address-cells' and '#size-cells' properties
definition for incoming possible panel child node.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit ccaa041b96a3124521f02c1e3d223890e659433b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index 0e2c16f..4fb1b97 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -1009,6 +1009,8 @@
 	};
 
 	mipi_dsi: mipi_dsi@32E10000 {
+		#address-cells = <1>;
+		#size-cells = <0>;
 		compatible = "fsl,imx8mm-mipi-dsim";
 		reg = <0x0 0x32e10000 0x0 0x400>;
 		clocks = <&clk IMX8MM_CLK_DSI_CORE_DIV>,
-- 
1.7.9.5

